### DISCUSSION TITLE
Detect failed login from postal-web

### DISCUSSION THREAD

**Discussion by dinodipardo** [postalserver/postal#2022] [Help with using Postal]
Hi, to implement fail2ban on the postal web component we are looking to detect a failed login attempt in the rails.log file.  However we do not see much usefull information to create a fail2ban filter.

Are there any settings we can tweak to get more logging for failed login attempts?

**Comment by willpower232** [comment]
I don't think such a thing has been implemented before. I imagine you would have to do something like this https://dradisframework.com/academy/knowledge-base/ruby/ruby-on-rails/protect-rails-application-with-fail2ban.html to complete the setup.

Alternatively, if you know the IP addresses that should be accessing the web UI, you could restrict it via firewall.
